### PR TITLE
[GITHUB-8] Update to Latest Standards

### PR DESCRIPTION
**Pull request by dennisconrad**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-# Options are standard or json
-sansible_nginx_access_log_format: json
+sansible_nginx_access_log_format: json  # either 'standard' or 'json'
 sansible_nginx_gzip: "on"
 sansible_nginx_gzip_comp_level: 2
 sansible_nginx_gzip_min_length: 1100
 sansible_nginx_gzip_static: "on"
 sansible_nginx_gzip_types: |
-    text/plain text/css application/x-javascript \
+    text/plain text/css application/x-javascript
```
Comment (lobsterdore) [inline defaults/main.yml]: Does this definitely produce the desired string without the backslash?

File: .version [hunk 2]
```diff
@@ -1 +1 @@
-v2.0
+v2.1
```
Comment (lobsterdore) [inline .version:1]: Need to update README to mention this version (v2.1).

File: defaults/main.yml (hunk 1 above)
Comment (dennisconrad) [reply defaults/main.yml]: No, I forgot to change the `|` to `>`

File: .version (hunk 2 above)
Comment (dennisconrad) [reply .version:1]: Thank you.

**Review by rsgoodall — APPROVED** [review]

**Pull request merged**